packages/jsii/lib/assembler.ts

```diff
@@ -345,6 +348,15 @@ export class Assembler implements Emitter {
     return [jsiiType];
   }
 
+  private declarationLocation(node: ts.Declaration): spec.SourceLocation {
+    const file = node.getSourceFile();
+    const line = ts.getLineAndCharacterOfPosition(file, node.getStart()).line;
+    return {
+      filename: path.relative(this.projectInfo.projectRoot, path.resolve(file.fileName)),
```

Comment:
Is the `path.resolve` necessary here?

Reply:
I don't know. I added it for safety. Guess I should put one around `projectRoot` too.

Reply:
Was asking mostly for curiosity :)

Reply:
I tried it out and it's actually not necessary!